apps/server/src/shared/domain/entity/video-conference.entity.ts

```diff
@@ -4,6 +4,8 @@ import { BaseEntityWithTimestamps } from './base.entity';
 export enum TargetModels {
 	COURSES = 'courses',
 	EVENTS = 'events',
+	ROOMS = 'rooms',
+	VIDEO_CONFERENCE_ELEMENTS = 'video-conference-elements',
```

Comment:
I also found myself in that rabbit hole when I thought about the naming. I thought about `BBB_ELEMENTS` but that was little to precise and not fitting with the rest of the services for VC. Otherwise something like `BOARD_ELEMENTS` could fit, but that would maybe be little too general. Whats your opinion on that?

Reply:
Looks like we should stick to `'video-conference-elements'`. Otherwise it would be too unspecific.

Reply:
I agree.